### PR TITLE
Fixed the link for create a ConfigMap

### PR DESCRIPTION
**Pull request by thisisharrsh**
Fixes: #45059

### CODE REVIEW

**Comment by thisisharrsh** [comment]
Deployment preview: https://deploy-preview-45060--kubernetes-io-main-staging.netlify.app/docs/concepts/storage/volumes/#configmap

File: content/en/docs/concepts/storage/volumes.md [hunk 1]
```diff
@@ -194,8 +194,7 @@ keyed with `log_level`.
 
 {{< note >}}
 
-* You must create a [ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/)
-  before you can use it.
+* You must [create a ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/#create-a-configmap) before you can use it.
```
Comment (stmcginnis) [inline content/en/docs/concepts/storage/volumes.md:197]: Would be nice to keep the line break so the line isn't so long. Makes it a little easier to read with editors that don't have line wrapping on. No difference in the rendered output, of course, so just a nit.

**Review by stmcginnis — COMMENTED** [review]
/lgtm

File: content/en/docs/concepts/storage/volumes.md (hunk 1 above)
Comment (thisisharrsh) [reply content/en/docs/concepts/storage/volumes.md]: I have updated the file with the suggested changes.

**Review by Princesso — COMMENTED** [review]
/lgtm

**Pull request merged**